be/src/vec/functions/function_string.cpp

```diff
@@ -265,6 +265,45 @@ struct TransferImpl {
     }
 };
 
+// Capitalize first letter
+struct NameToInitcap {
+    static constexpr auto name = "initcap";
+};
+
+struct InitcapImpl {
+    static Status vector(const ColumnString::Chars& data, const ColumnString::Offsets& offsets,
+                         ColumnString::Chars& res_data, ColumnString::Offsets& res_offsets) {
+        size_t offset_size = offsets.size();
+        res_offsets.resize(offsets.size());
```

Comment:
here maybe need memcpy， the offset do not change

Reply:
done.